msu/hooks/entity/tactical/tactical_entity_manager.nut

```diff
@@ -1,4 +1,27 @@
 ::mods_hookNewObject("entity/tactical/tactical_entity_manager", function(o) {
+	// VANILLAFIX: http://battlebrothersgame.com/forums/topic/oncombatstarted-is-not-called-for-ai-characters/
+	local spawn = o.spawn;
+	o.spawn = function( _properties )
+	{
+		local ret = spawn(_properties);
+		foreach (i, faction in this.getAllInstances())
+		{
+			if (i != ::Const.Faction.Player)
+			{
+				foreach (actor in faction)
+				{
+					actor.getSkills().onCombatStarted();
+					actor.getItems().onCombatStarted();
+					actor.getSkills().update();
+				}
+			}
+		}
+
+		::Math.seedRandom(::Time.getRealTime());
```

Comment:
Not sure why we need to do this since the end of the spawn function already takes care of re-randomizing the seed in vanilla.

Reply:
I didn't know what was the purpose of that line at the end of the vanilla function, so I just added it here to be safe in case its relevant for something. Should I remove it?